.env.example

```
@@ -96,3 +96,8 @@ SMTP_PASSWORD=
 EMAILS_FROM_EMAIL=
 EMAILS_FROM_NAME=Kaapi
 FRONTEND_HOST=
+
+# AWS Secrets Manager
+USE_AWS_SECRETS=false
+AWS_SECRETS_REGION=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AWS_SECRETS_REGION key should go before the USE_AWS_SECRETS key

(UnorderedKey)